docs/part1.md

```diff
@@ -418,12 +414,12 @@ For your answer, **take a look at the side menu**.
         </label>
     </div>
 
-    In practice, when doing **machine learning**, you would only choose a **single functional** (HSE06 in this case).
-    The only exception would be _transfer learning_, but even then you are advised to split the functionals out over separate queries.
-    Here, we look for 2 functionals just for educational reasons.
+    <!-- In practice, when doing **machine learning**, you would only choose a **single functional** (HSE06 in this case).
+    The only exception would be _transfer learning_, but even then you are advised to split the functionals out over separate queries. -->
+    Note: In practice you would only choose a **single functional** when doing **machine learning**. Here, we look for 2 functionals just for educational reasons.
 
 ???+ info "Multiple density functionals per entry"
-    A single entry (and even calculation) may contain multiple functional names, just not `XC`!
+    A single entry (and even calculation) may contain multiple functional names, not just `XC`!
```

Comment:
No, the original statement is correct: a functional either contains 1 XC, or a X and C.
As with anything that people come up with, there are other (wilder) combinations out there, but they're fairly uncommon.
We could also just leave out the `XC` specification at the end. Either way, I doubt this audience will care much.

Reply:
oh sorry, I just misread it. You can put it back it's fine